### ISSUE TITLE
Some NHK Live streams have stopped working

### ISSUE THREAD

**Issue by fearlesschicken** [streamlink/streamlink#2736]
## Issue
- [x] This is a plugin issue and I have read the contribution guidelines.

### Description

NHK live no longer works with some of the available streams.  Only the "###p_alt" streams work.

### Expected / Actual behavior

All streams should be working.

### Logs
```
streamlink https://www3.nhk.or.jp/nhkworld/en/live/ 720p

[cli][info] Found matching plugin nhkworld for URL https://www3.nhk.or.jp/nhkworld/en/live/
[cli][info] Available streams: 180p_alt2 (worst), 180p_alt, 180p, 360p_alt2, 360p_alt, 360p, 720p_alt, 720p (best)
[cli][info] Opening stream: 720p (hls)
[cli][error] Try 1/1: Could not open stream <HLSStream('https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_1180.m3u8')> (Could not open stream: Unable to open URL: https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_1180.m3u8 (404 Client Error: Not Found for url: https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_1180.m3u8))
error: Could not open stream <HLSStream('https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_1180.m3u8')>, tried 1 times, exiting

streamlink https://www3.nhk.or.jp/nhkworld/en/live/ 720p_alt

[cli][info] Found matching plugin nhkworld for URL https://www3.nhk.or.jp/nhkworld/en/live/
[cli][info] Available streams: 180p_alt2 (worst), 180p_alt, 180p, 360p_alt2, 360p_alt, 360p, 720p_alt, 720p (best)
[cli][info] Opening stream: 720p_alt (hls)
[cli][info] Starting player: mpv --cache 2048

streamlink https://www3.nhk.or.jp/nhkworld/en/live/ 360p

[cli][info] Found matching plugin nhkworld for URL https://www3.nhk.or.jp/nhkworld/en/live/
[cli][info] Available streams: 180p_alt2 (worst), 180p_alt, 180p, 360p_alt2, 360p_alt, 360p, 720p_alt, 720p (best)
[cli][info] Opening stream: 360p (hls)
[cli][error] Try 1/1: Could not open stream <HLSStream('https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_900.m3u8')> (Could not open stream: Unable to open URL: https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_900.m3u8 (404 Client Error: Not Found for url: https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_900.m3u8))
error: Could not open stream <HLSStream('https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp/index_900.m3u8')>, tried 1 times, exiting

streamlink https://www3.nhk.or.jp/nhkworld/en/live/ 360p_alt

[cli][info] Found matching plugin nhkworld for URL https://www3.nhk.or.jp/nhkworld/en/live/
[cli][info] Available streams: 180p_alt2 (worst), 180p_alt, 180p, 360p_alt2, 360p_alt, 360p, 720p_alt, 720p (best)
[cli][info] Opening stream: 360p_alt (hls)
[cli][info] Starting player: mpv --cache 2048





```

**Comment by fearlesschicken** [comment]
As of today, the alt streams are also broken :(

```streamlink https://www3.nhk.or.jp/nhkworld/en/live/ 720p_alt
[cli][info] Found matching plugin nhkworld for URL https://www3.nhk.or.jp/nhkworld/en/live/
[cli][info] Available streams: 180p_alt2 (worst), 180p_alt, 180p, 360p_alt2, 360p_alt, 360p, 720p_alt, 720p (best)
[cli][info] Opening stream: 720p_alt (hls)
[cli][error] Try 1/1: Could not open stream <HLSStream('https://b-nhkwlive-xjp.akamaized.net/hls/live/2003458-b/nhkwlive-xjp/index_1180.m3u8')> (Could not open stream: Unable to open URL: https://b-nhkwlive-xjp.akamaized.net/hls/live/2003458-b/nhkwlive-xjp/index_1180.m3u8 (404 Client Error: Not Found for url: https://b-nhkwlive-xjp.akamaized.net/hls/live/2003458-b/nhkwlive-xjp/index_1180.m3u8))
error: Could not open stream <HLSStream('https://b-nhkwlive-xjp.akamaized.net/hls/live/2003458-b/nhkwlive-xjp/index_1180.m3u8')>, tried 1 times, exiting```

**Comment by beardypig** [comment]
Fixed in #2733
